addons/sys_bf888s/radio/fnc_getExternalAudioPosition.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ *
+ * Arguments:
+ * 0: Radio ID <STRING> (Unused)
```

Comment:
Is it really unused ? It's reffed in the code below, so I suspect it's an accidental copypaste.